example_project/person.py

```diff
@@ -24,3 +24,8 @@ def __repr__(self):
         return util.format_repr(
             self.__class__.__qualname__, name=self.name, age=self.age
         )
+
+
+class Student(Person):
```

Comment:
Adding an overload constructor `__init__` with an additional parameter might be useful. Call `Person.__init__` inside it and see what happens.

For example,

```py
class Student(Person):
    def __init__(self, name, age, sid):
        Person.__init__(self, name, age)
        self.id = sid
```